.tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "jsx": "react",
+    "pretty": true,
+    "skipLibCheck": true,
+    "allowJs": true
```

Comment:
Maybe enable strict?
Will result in more errors initially, but worth for the long run.

Reply:
good catch! it looks like it caught 3ish more errors

Reply:
@ksjogo just fixed in https://github.com/drcmda/react-three-fiber/pull/86!